grpclib/client.py

```diff
@@ -792,11 +793,14 @@ def __del__(self) -> None:
         if self._protocol is not None:
             message = 'Unclosed connection: {!r}'.format(self)
             warnings.warn(message, ResourceWarning)
-            if self._loop.is_closed():
```

Comment:
Placing `assert self._loop is not None` before using `self._loop` property will do the trick for mypy, this is probably the simplest way of telling to mypy that variable was initialised. If `self._protocol is not None` then also `self._loop is not None`.